project/Dependencies.scala

```diff
@@ -53,6 +56,7 @@ object Dependencies {
   val sprayJson     = "io.spray"        %% "spray-json"    % "1.3.1"
 
   val apacheMath    = "org.apache.commons" % "commons-math3" % "3.2"
+  val scalaBreeze   = "org.scalanlp"  %%  "breeze"  % "0.11.2"
```

Comment:
What exactly are we using this dependency for? Is there a way to not use it? It's a big enough dependency that I want to be sure we need it to add it to the project.